### PR TITLE
add functions for getting metadata about a specific person

### PR DESCRIPTION
**Pull request by BobBorges**
dev

### CODE REVIEW

File: pyriksdagen/utils.py [hunk 1]
```diff
@@ -26,6 +26,10 @@
 XML_NS = "{http://www.w3.org/XML/1998/namespace}"
 TEI_NS = "{http://www.tei-c.org/ns/1.0}"
 
+def fetch_ns():
+    return {"tei_ns": "{http://www.tei-c.org/ns/1.0}",
```
Comment (ninpnin) [inline pyriksdagen/utils.py:30]: Use the lines above?

**Review by ninpnin — CHANGES_REQUESTED** [review]

File: pyriksdagen/utils.py (hunk 1 above)
Comment (BobBorges) [reply pyriksdagen/utils.py:30]: I use namespaces as a dict object all over the project. Would you be happy with a compromise?

```
def fetch_ns():
    return {"tei_ns": TEI_NS,
            "xml_ns": XML_NS}
```

Comment (BobBorges) [reply pyriksdagen/utils.py:30]: image
image

Comment (BobBorges) [reply pyriksdagen/utils.py]: https://github.com/swerik-project/pyriksdagen/pull/15/commits/eaf1f83c3ed935c7cd77535abca123abdfcb3348

**Review by ninpnin — APPROVED** [review]

**Pull request merged**